Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -137,6 +155,30 @@ static string RemoveHyphenAndUnderscore(string stringValue)
                         p => p.JsonFieldName,
                         p => new KeyValuePair<Type, ReflectionUtils.SetDelegate>(p.Type, p.SetDelegate));
             }
+
+            private static Type GetPayloadType(string activityType)
```

Comment:
Personally, I'd prefer to keep it how it is - but I'm happy to change it to an extension method if you want.